src/operator/l2_normalization-inl.h

```diff
@@ -294,7 +321,13 @@ class L2NormalizationProp : public OperatorProperty {
     return {ResourceRequest::kTempSpace};
   }
 
-  Operator* CreateOperator(Context ctx) const override;
+  Operator* CreateOperator(Context ctx) const override {
```

Comment:
Does something still call this?

Reply:
Honestly I'm not really sure, with a simply grep for "CreateOperator" in src only this usage appeared:
nnvm/legacy_op_util.cc:297:  return OpStatePtr::Create<OperatorState>(prop.ptr->CreateOperatorEx(ctx, &is, &it),

Reply:
Ok, I see it is masked by your override of CreateOperatorEx()